File: agitr/set_bg_color.cpp
#include <ros/ros.h>
#include <std_srvs/Empty.h>
int main(int argc, char **argv){
  // initalize the ROS system
  ros::init(argc, argv, "set_bg_color");
  
  // establish this program as a ROS Node
  ros::NodeHandle nh;



  ros::service::waitForService("clear");

  ros::param::set("background_r",255);
  ros::param::set("background_g",255);
  ros::param::set("background_b",0);


  ros::ServiceClient clearClient = nh.serviceClient<std_srvs::Empty>("/clear");

  std_srvs::Empty srv;

  clearClient.call(srv);
}

File: pyagitr/scripts/set_bg_color.py
#!/usr/bin/env python

import rospy #include <ros/ros.h> cpp equivalent
from std_srvs.srv import Empty

def setBgColorLoop():
    rospy.init_node('set_bg_color')

    rospy.wait_for_service("clear")

    rospy.set_param("/turtlesim/background_r",255);
    rospy.set_param("/turtlesim/background_g",255);
    rospy.set_param("/turtlesim/background_b",0);

    #ros::ServiceClient clearClient = nh.serviceClient<std_srvs::Empty>("/clear");
    #std_srvs::Empty srv;

    clearClient = rospy.ServiceProxy('/clear', Empty)

    clearClient()

if __name__ == '__main__':
    try:
        setBgColorLoop()
    except rospy.ROSInterruptException:
        pass
